src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -41,6 +41,9 @@
 ### Behavior Changes
 - As a part of the transition to pandas 2.2.1, pandas `df.loc` and `__setitem__` have buggy behavior when a column key is used to assign a DataFrame item to a DataFrame (a scalar column key and DataFrame item are used for assignment (https://github.com/pandas-dev/pandas/issues/58482)). Snowpark pandas deviates from this behavior and will maintain the same behavior as pandas from versions 1.5.x.
 
+### New Features
+Added partial support for `SeriesGroupBy.apply`.
```

Comment:
I think it would be good to call out the specifics of the parameters supported for SeriesGroupBy.apply.